### PR TITLE
Document how to use springdoc-openapi

### PR DESCRIPTION
**Pull request by maexled**
Part of Gamify-IT/issues#163

### CODE REVIEW

**Review by delvh — APPROVED** [review]

**Review by Gr33ndev — APPROVED** [review]

**Review by Aaron2000119 — APPROVED** [review]

**Review by cdingm — APPROVED** [review]

File: dev-manuals/languages/java/springdoc.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  <dependency>
+		<groupId>org.springdoc</groupId>
+		<artifactId>springdoc-openapi-ui</artifactId>
+		<version>${org.springdoc.version}</version>
+	</dependency>
+  ...
+</dependencies>
+
+```
+Also add the path of the page in `application.propteries`:
```
[the remark this answers is not in the document]
Comment (maexled) [reply dev-manuals/languages/java/springdoc.md:23]: I think everyone should find this file when working with spring-boot

[the remark this answers is not in the document]
Comment (delvh) [reply dev-manuals/languages/java/springdoc.md:23]: Ah, I think you misunderstood what I meant:
I meant the `server.servlet...` key, not the path of the `application.properties`...

[the remark this answers is not in the document]
Comment (maexled) [reply dev-manuals/languages/java/springdoc.md:23]: That has nothing to do with springdoc

[the remark this answers is not in the document]
Comment (delvh) [reply dev-manuals/languages/java/springdoc.md:23]: Yes, it should simply be noted _somewhere_.
Do you have a better suggestion where to mention that?

[the remark this answers is not in the document]
Comment (maexled) [reply dev-manuals/languages/java/springdoc.md:23]: maybe extra markdown file for general spring configurations

**Pull request merged**